examples/ovs-cni-marker.yml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
I find it confusing that you provide two versions of manifests for marker, one that includes the new container into ovs-cni daemon set, and one that ships it separately. It makes more sense to leave one that you believe is the best approach to deploy it.